src/streamlink/plugins/cnews.py

```diff
@@ -1,26 +1,24 @@
 import re
 
 from streamlink.plugin import Plugin
-from streamlink.plugin.api import http, useragents, validate
-from streamlink.stream import HDSStream, HLSStream, HTTPStream
+from streamlink.plugin.api import http, useragents
 
 
 class CNEWS(Plugin):
-    _url_re = re.compile(r'(https|http)://www.cnews.fr(/\w)+')
+    _url_re = re.compile(r'https?://www.cnews.fr(/\w)+')
```

Comment:
Looks like there is a mistake in the regex... You have a repeating group of `/\w` so it will only match a single character after a `/` (repeated) eg. `/f`, `/f/x`, `/f/x/w/x/t/t`.. which is probably not what you intended :) 
Since you are using `re.match` for the URL test, and if you want to match any page on `cnews` you could do `r'https?://www.cnews.fr/'` and it'll match everything.

Reply:
I didn't see you tests just now :) 

I see that you don't want to match the root path. I would do r'https?://www.cnews.fr/.+', which should match any non-root path on the site.